umh-core/pkg/service/filesystem/filesystem.go

```diff
@@ -457,6 +683,9 @@ func (s *DefaultService) RemoveAll(ctx context.Context, path string) error {
 			return fmt.Errorf("failed to remove directory %s: %w", path, err)
 		}
 
+		// Invalidate cache entries for this path and all subpaths
```

Comment:
Could make sense to also invalidate on error, otherwise we might be in an unknown state